### DISCUSSION TITLE
Pull requesting review etiquette

### DISCUSSION THREAD

**Discussion by KerfuffleV2** [ggml-org/llama.cpp#1566] [Q&A]
When you create a pull request, GitHub's interface suggests several people that could be requested to review it. Is it a good idea/polite to actually use that, or is it better to just wait for someone to see the pull and deal with it "naturally"?

I haven't used that feature so far, because it feels like it could seem like saying "Hey, deal with my pull right now!" Presumably the people who want to review pull requests will check the list of recent new pull requests periodically when they have time and don't need to be reminded but perhaps that assumption is incorrect.

Thanks for reading.

**Comment by Green-Sky** [comment]
you can request me when its cmake, github ci or maybe c++.
no guaranty i will actually look at it. but i would appreciate it if i get notified of cmake and ci changes :)
